Maquetado/index.html

```diff
@@ -12,6 +12,19 @@
 
 <body>
   <h2 class="title">Front-End Tools</h2>
-</body>
 
+  <div class="tech">
```

Comment:
for this feature we just need to add one card without any layout related stuff so you can delete the div with class row, the div with class column and the div with class square. just use this div to add the green background and a `::before` pseudo-element to create the green border.